### DISCUSSION TITLE
The command "kimai:reset:locales" does not exist

### DISCUSSION THREAD

**Discussion by hyouhaku** [kimai/kimai#3843] [Questions & Answers]
Finally got Azure SAML working
However I'm receiving "An authentication exception occurred." error when trying to login with Azure AD
Upon checking prod.log, looks like it's trying to reset locale when creating super user, and error message looks like this:
`console.CRITICAL: Error thrown while running command "'kimai:reset:locales'". Message: "The command "kimai:reset:locales" does not exist." {"exception":"[object] (Symfony\\Component\\Console\\Exception\\CommandNotFoundException(code: 0): 
The command \"kimai:reset:locales\" does not exist. at /opt/kimai/vendor/symfony/console/Application.php:551)","command":"'kimai:reset:locales'","message":"The command \"kimai:reset:locales\" does not exist."} []`
build kimai/kimai2:apache-2.0.0-prod

**Comment by kevinpapst** [comment]
> Finally got Azure SAML working

Great, share your steps in the other discussion

The command `kimai:reset:locales` can only be called manually via bash and is disabled in `prod` environments.
Please stay in the other discussion.
